Bounding_volumes/doc/Bounding_volumes/CGAL/Min_sphere_of_spheres_d_traits_d.h

```diff
@@ -38,7 +38,7 @@ class Min_sphere_of_spheres_d_traits_d {
 /*!
 is the constant `Dim`. 
 */ 
-typedef unspecified_type D; 
+static const int D;
```

Comment:
Actually that seems like another doc problem: `Dim` is a template parameter documented as `typename Dim`, but it seems to me that it should be `int Dim`. Well, actually I checked and this is exactly what is done in the include file…

```c++
  template<typename K_,                      // kernel
    typename FT_,                            // number type
    int Dim_,                                // dimension
    typename UseSqrt_ = Tag_false,           // whether to use square-roots
    typename Algorithm_ = Farthest_first_heuristic> // algorithm to use
  class Min_sphere_of_spheres_d_traits_d {
  public: // types:
    typedef FT_ FT;
    typedef FT_ Radius;
    typedef typename K_::Point_d Point;
    typedef std::pair<Point,Radius> Sphere;
    typedef typename K_::Cartesian_const_iterator_d Cartesian_const_iterator;
    typedef UseSqrt_ Use_square_roots;
    typedef Algorithm_ Algorithm;

  public: // constants:
    static const int D = Dim_;               // dimension
```

Here again, I would be tempted to simply correct the documentation and replace `typename Dim` by `int Dim` which is exactly what the code is.

(At some point, we should really get rid of these separated documentation header files, I know this is something inherited from old doc generation of CGAL, but it is really error-prone.)